common/content/commandline.js

```diff
@@ -421,6 +421,16 @@ const CommandLine = Module("commandline", {
         }
     },
 
+    _isIMEComposing: false,
```

Comment:
maybe this should be called something slightly more general?
While it does mostly goes to fix IME input, it could be a useful property to have and I don't think it is actually set when IME is active.